### PR TITLE
Add example with two layer on y axis

### PR DESCRIPTION
**Pull request by ThomAub**
This implement #7

### CODE REVIEW

**Comment by mockersf** [comment]
Could you add your example to the README.md in examples (use https://tinypng.com to ensure image is small)

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,9 @@
 # Created by https://www.gitignore.io/api/rust,jupyternotebooks
 # Edit at https://www.gitignore.io/?templates=rust,jupyternotebooks
 
+### VScode ###
+.vscode/
+
```
Comment (davidB) [inline .gitignore]: see the top comment, .gitignore is generated. As gitignore.io is able to also generate code for vscode. Can you update gitignore with what is generated by https://www.gitignore.io/api/rust,jupyternotebooks,visualstudiocode

File: .gitignore [hunk 2]
```diff
@@ -2,8 +2,7 @@
 # Created by https://www.gitignore.io/api/rust,jupyternotebooks
 # Edit at https://www.gitignore.io/?templates=rust,jupyternotebooks
 
-### VScode ###
-.vscode/
+/
```
Comment (mockersf) [inline .gitignore]: not sure it's a good idea / useful to add `/` to .gitignore

Comment (davidB) [reply .gitignore]: Just copy the full output of the link I provided (including update of comment,...) and keep the order, so next update will be simpler (no manual merge,...)

Comment (ThomAub) [reply .gitignore]: copy/pasted the generated https://www.gitignore.io/api/rust,jupyternotebooks,visualstudiocode

File: .gitignore (hunk 1 above)
Comment (ThomAub) [reply .gitignore]: done below

**Review by mockersf — APPROVED** [review]

File: examples/line_with_interval.rs [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        )
+        .layer(vec![
+            LayerSpecBuilder::default()
+                .mark(Mark::Line)
+                .encoding(
+                    LayerEncodingBuilder::default()
+                        .y(YClassBuilder::default()
+                            .field("Miles_per_Gallon")
+                            .def_type(StandardType::Quantitative)
+                            .aggregate(Aggregate::Enum(AggregateOp::Mean))
```
Comment (mockersf) [inline examples/line_with_interval.rs]: this can now be written as `.aggregate(AggregateOp::Mean)`

File: examples/line_with_interval.rs [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+                            .build()?)
+                        .build()?,
+                )
+                .build()?,
+            LayerSpecBuilder::default()
+                // .mark(Mark::Errorband)
+                .mark(AnyMark::MarkDefClass(
+                    MarkDefClassBuilder::default()
+                        .def_type(Mark::Errorband)
+                        .extent(BoxPlotDefExtent::Enum(ExtentExtent::Ci))
```
Comment (mockersf) [inline examples/line_with_interval.rs]: same here, wrapper enum can be removed to `.extent(ExtentExtent::Ci)`

**Pull request merged**